packages/medusa/src/services/product-category.ts

```diff
@@ -65,6 +65,30 @@ class ProductCategoryService extends TransactionBaseService {
 
     return productCategoryTree
   }
+
+  /**
+   * Deletes a product category
+   *
+   * @param productCategoryId is the id of the product category to delete
+   * @return a promise
+   */
+  async delete(productCategoryId: string): Promise<void> {
```

Comment:
todo: We have convention to do idempotent deletes in Medusa, if the product category is not found we shouldn't throw and just return a success